packages/x-components/src/x-modules/tagging/store/module.ts

```diff
@@ -26,9 +27,7 @@ export const taggingXStoreModule: TaggingXStoreModule = {
     setQueryTaggingInfo(state, queryTaggingInfo: TaggingRequest) {
       state.queryTaggingInfo = queryTaggingInfo;
     },
-    setTaggingConfig(state, taggingConfig) {
-      Object.assign(state.config, taggingConfig);
-    }
```

Comment:
Removing this mutation (or renaming it) is a breaking change. You have to document it in the PR description and [in the commit message](https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-description-and-breaking-change-footer)